<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectFolder</class>
 <widget class="QDialog" name="SelectFolder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>80</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>80</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,0">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton_ok">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="text_curFile">
       <property name="text">
        <string>Current folder:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_currentFolder">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_selFolder">
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/Images/folder.png</normaloff>:/Images/folder.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
